app/Config/Schema/sessions.php

```diff
@@ -24,34 +25,35 @@
  * cake schema run create Sessions
  *
  */
-class SessionsSchema extends CakeSchema {
-
-	public $name = 'Sessions';
-
-/**
- * Before callback.
- *
- * @param array $event Schema object properties
- * @return bool Should process continue
- */
-	public function before($event = array()) {
-		return true;
-	}
+class SessionsSchema extends CakeSchema
```

Comment:
__[staging][code_sniffer]__
* You must use "/**" style comments for a class comment